app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
```

Comment:
- [ ] As per the given _ERD_, establishing the _Post_ **association** should be such that _Post_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

Let's also note that this association is **not optional**